### ISSUE TITLE
Opening Settings -> Tags error

### ISSUE THREAD

**Issue by Jazekss** [files-community/Files#11400]
### Description

When try to open in settings section of Tags its close app, first time its give error with text: Something went wring and thats all, but i think this error message give Windows, not Files app

### Steps To Reproduce

1. Go to Settings
2. Try to open Tags section
3. App close

### Steps to fix

- eg. Fix crash that occurs when...
- eg. Correct action happens when clicking...

### Files Version

Version 2.4.40.0

### Windows Version

Windows 10 Pro N 21h2

### Log file

[debug.log](https://github.com/files-community/Files/files/10797242/debug.log)

**Comment by Josh-65** [comment]
Thanks for the report I have change the title and steps to fix to better reflect the issue.

**Comment by yaira2** [comment]
Have you made modifications to the tags file before the tags UI was released?

**Comment by ferrariofilippo** [comment]
@yair100 can you close #11394? It's a duplicate of this one.
There might be an issue with nulls in the context menu too, but I can easily fix that. However, I'm not sure why this settings bug is happening

**Comment by ferrariofilippo** [comment]
@Jazekss can you upload `filetags.json`? You can find it under `settings` folder when you click `Open Log Location`

**Comment by yaira2** [comment]
@ferrariofilippo unless someone edits the file, is there any case where the value would be null?

**Comment by ferrariofilippo** [comment]
I don't think so. Anyway, I could add a null-check to `ColorHelpers.FromHex()` and a `IsColorValid` method to check tags each time they are loaded

**Comment by Jazekss** [comment]
> Have you made modifications to the tags file before the tags UI was released?

Yes, write colors and names by hand  i am designer and i know how hex colors work, write alk properly, simply change name and color code, can upload file now, only later, i change that colors at first use Files and all works correctly until the day before yesterday, two day no errors

**Comment by Jazekss** [comment]
> I don't think so. Anyway, I could add a null-check to `ColorHelpers.FromHex()` and a `IsColorValid` method to check tags each time they are loaded

I thin this will be grate and escape errors like that
